airbyte-integrations/connectors/source-instatus/source_instatus/instatus.yaml

```diff
@@ -0,0 +1,286 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: [ ]
+
+  user_field_selector:
+    extractor:
+      field_pointer: [ "user" ]
+
+  requester:
+    url_base: "https://api.instatus.com/"
+    http_method: "GET"
+    authenticator:
+      type: BearerAuthenticator
+      api_token: "{{ config['api_key'] }}"
+
+  public_data_requester:
```

Comment:
It looks like the `public_data_requester` is duplicating almost everything from the `requester`/`api_error_handler_requester`. The main thing that appears to change is that the url_base is set to `https://{{ stream_slice.url }}.instatus.com/summary.json` instead. Can we instead refactor this a bit so that everything can be defined on `requester` and reduce the duplicated fields.. 

The `public_data_requester` can just be written to override the one `url_base` instead of the whole thing:

```
public_data_requester:
  $ref: "*ref(definitions.requester)"
  url_base: "https://{{ stream_slice.url }}.instatus.com/summary.json"
```

Reply:
done